### ISSUE TITLE
Versioned graph url doesn't work with exchanging long lived access token

### ISSUE THREAD

**Issue by gr8bit** [mweibel/facebook.ex#56]
I want to pin the API version my app will be using. To do so, I set the `graph_url` with Facebook's versioning scheme to a specific version, like this one: `https://graph.facebook.com/v2.11/`.
This, however, breaks the `long_lived_access_token` call as the uri used therein (`/oauth/access_token`) isn't available in versioned paths, only at root.
Maybe the config could introduce a parameter `graph_api_version` which will be appended only to the calls which are available under versioned urls. What do you think?

**Comment by mweibel** [comment]
Sounds like a good idea! I'd welcome a PR 👍

**Comment by gr8bit** [comment]
I feared that you'd say that ;) I'll definitely look into that asap but I'm quite with other project atm.

**Comment by gr8bit** [comment]
I started implementing it yesterday and came to an embarrassing conclusion: it was my fault. To use versioned graph api urls, you actually can append the version like you intended - but ONLY if you don't append a trailing slash!
`https://graph.facebook.com/v2.11` works fine
`https://graph.facebook.com/v2.11/` fails with some Facebook error saying that some path components are unknown.
So I changed to the non-slash variant and my long lived access token call went through :)

Sorry for crying wolf ;)
